phpBB/styles/prosilver/template/posting_pm_header.html

```diff
@@ -1,83 +1,110 @@
 <fieldset class="fields1">
-	<!-- IF not S_SHOW_DRAFTS -->
+	{% if not S_SHOW_DRAFTS %}
 
-		<!-- IF S_GROUP_OPTIONS -->
+		{% if S_GROUP_OPTIONS %}
 			<div class="column2">
-				<label for="group_list"><strong>{L_TO_ADD_GROUPS}{L_COLON}</strong></label><br />
-				<select name="group_list[]" id="group_list" multiple="multiple" size="3" class="inputbox">{S_GROUP_OPTIONS}</select><br />
+				<label for="group_list"><strong>{{ lang('TO_ADD_GROUPS') }}{{ lang('COLON') }}</strong></label><br />
+				<select name="group_list[]" id="group_list" multiple="multiple" size="3" class="inputbox">{{ S_GROUP_OPTIONS }}</select><br />
 			</div>
-		<!-- ENDIF -->
-		<!-- IF S_ALLOW_MASS_PM -->
-		<div class="column1">
-			<!-- IF not S_EDIT_POST -->
-			<dl class="pmlist">
-				<dt><label><strong>{L_TO_ADD_MASS}{L_COLON}</strong><textarea id="username_list" name="username_list" class="inputbox" cols="50" rows="2" tabindex="1"></textarea></label></dt>
-				<dd class="recipients">
-				<input type="submit" name="add_to" value="{L_ADD}" class="button2" tabindex="1" />
-				<input type="submit" name="add_bcc" value="{L_ADD_BCC}" class="button2" tabindex="1" />
-				<!-- EVENT posting_pm_header_find_username_before -->
-				<span><a href="{U_FIND_USERNAME}" onclick="find_username(this.href); return false;">{L_FIND_USERNAME}</a></span>
-				<!-- EVENT posting_pm_header_find_username_after -->
-				</dd>
-			</dl>
-			<!-- ENDIF -->
-		</div>
-		<!-- IF .to_recipient or .bcc_recipient --><hr /><!-- ENDIF -->
-		<div class="column1">
-			<!-- IF .to_recipient -->
-				<dl>
-					<dt><label>{L_TO_MASS}{L_COLON}</label></dt>
+		{% endif %}
+		{% if S_ALLOW_MASS_PM %}
+			<div class="column1">
+				{% if not S_EDIT_POST %}
+				<dl class="pmlist">
+					<dt><label><strong>{{ lang('TO_ADD_MASS') }}{{ lang('COLON') }}</strong><textarea id="username_list" name="username_list" class="inputbox" cols="50" rows="2" tabindex="1"></textarea></label></dt>
 					<dd class="recipients">
-					<ul class="recipients">
-						<!-- BEGIN to_recipient -->
-						<li>
-							<!-- IF not S_EDIT_POST --><input type="submit" name="remove_{to_recipient.TYPE}[{to_recipient.UG_ID}]" value="x" class="button2" /><!-- ENDIF -->
-							<!-- IF to_recipient.IS_GROUP --><a href="{to_recipient.U_VIEW}" style="color: {{ to_recipient.COLOUR }}"><strong>{to_recipient.NAME}</strong></a><!-- ELSE -->{to_recipient.NAME_FULL}<!-- ENDIF -->
-						</li>
-						<!-- END to_recipient -->
-					</ul>
+						<input type="submit" name="add_to" value="{{ lang('ADD') }}" class="button2" tabindex="1" />
```

Comment:
Void elements should be closed with a simple `>` instead of ` />` - applies to the whole file.